tests/benchmarks/benchmark.py

```diff
@@ -107,7 +107,7 @@ def test_DSHAPE_run(tmpdir_factory, benchmark):
 @pytest.mark.benchmark(min_rounds=1, max_time=300, disable_gc=True, warmup=False)
 def test_DSHAPE_current_run(tmpdir_factory, benchmark):
     """Benchmark the DSHAPE fixed toroidal current example."""
-    input_path = ".//tests//benchmarks//DSHAPE_current"
+    input_path = ".//tests//inputs//DSHAPE_current"
```

Comment:
Now that `tests/benchmarks` actually has these files, we should revert this change.

Reply:
I'd rather keep all the inputs in the `inputs` folder, I'll modify the benchmarks PR to use those same files so we don't have dupes.